x-pack/plugins/remote_clusters_management/index.js

```diff
@@ -12,6 +12,7 @@ import { registerListRoute } from './server/routes/api/remote_clusters';
 export function remoteClustersManagement(kibana) {
   return new kibana.Plugin({
     id: PLUGIN.ID,
+    configPrefix: 'xpack.remote_clusters',
```

Comment:
I am good with changing UI elements (URL, link text) to just `Remote Clusters`, but I think the config should stay as `xpack.remote_clusters_management` to differentiate the licensed management UI feature from OSS support